docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+FROM openjdk:11.0.2-jre-slim-stretch
```

Comment:
Can we update this to 11.0.3 as that appears to be the latest in this series

Reply:
done